data/pathfinder/paizo/player_companion/people_of_the_north/potn_equip_general.lst

```diff
@@ -1,4 +1,4 @@
-# CVS $Revision: $ $Author: $ -- Fri Jun 17 21:51:56 2016 -- reformated by PCGen PrettyLST v6.06.00
+# CVS $Revision: $ $Author: $ -- Fri Jan 29 20:50:52 2016 -- reformated by PCGen PrettyLST v6.06.00
```

Comment:
I didn't actually mean to include this file. It was part of an early clean-up I did to better see what fixes were needed to implement PFS. Sorry. I'll pull this!